### PR TITLE
Adds helm chart for express api.

### PR DESCRIPTION
**Pull request by rcstr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,12 @@ helm install --name ethereum-testnet transmute-charts/ganache
 ### ipfs
 
 ```
-helm install --name ethereum-testnet transmute-charts/ganache
+helm install --name ethereum-testnet transmute-charts/ipfs
+
+### ipfs
```
Comment (OR13) [inline README.md]: This should say api

File: api/values.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Default values for api.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
```
Comment (OR13) [inline api/values.yaml]: This image should be the docker image built for the node.js api with swagger.

Comment (OR13) [reply api/values.yaml]: maybe not, i misread this.

**Review by OR13 — CHANGES_REQUESTED** [review]
can we move all the api node.js dependencies under a root directory `api`, eventually it will need to be moved to its own package in the mono repo. for now, I'd like to keep it isolated from the rest of this charts work.

Please also add a travis test to show the chart deployment succeeds.

File: package.json [hunk 3]
```diff
@@ -1,14 +1,18 @@
 {
   "name": "transmute-charts",
   "version": "1.0.0",
+  "private": true,
```
Comment (eolszewski) [inline package.json]: @OR13 do we want these to be private?

**Review by OR13 — APPROVED** [review]

**Pull request merged**